packages/core/src/components/Dropdown/Dropdown.tsx

```diff
@@ -127,7 +128,12 @@ const Dropdown: VibeComponent<DropdownComponentProps, HTMLElement> & {
       return defaultValue;
     }, [defaultValue]);
 
+    BaseSelect.prototype.renderLiveRegion = () => {
```

Comment:
I needed to override this select function since it is adding a span with aria live that causing A11y bug